presto-verifier/src/main/java/com/facebook/presto/verifier/framework/VerificationContext.java

```diff
@@ -29,22 +30,25 @@
 
     private final int resubmissionCount;
     private final Set<QueryException> queryExceptions;
+    private final Set<ClientException> clientExceptions;
```

Comment:
The reason we have `queryExceptions` here in the `VerificationContext`, is that we eventually calls `VerificationContext.getQueryFailures` to gather the list of all exceptions seen during verification of a test cases, including the retries. Adding `clientExceptions` here but not using it is not helpful.

Instead, we should create a subclass of `QueryException`, and make `ClientException` the cause of it.

Reply:
Addressed it.

Reply:
This, along with this class, to be unchanged?

Reply:
yes removed the extra code